src/lib.rs

```diff
@@ -92,12 +92,23 @@ pub struct LegalOfficerCase<AccountId, Hash, LocId, BlockNumber> {
 
 pub type LegalOfficerCaseOf<T> = LegalOfficerCase<<T as frame_system::Config>::AccountId, <T as pallet::Config>::Hash, <T as pallet::Config>::LocId, <T as frame_system::Config>::BlockNumber>;
 
+pub type LicenseItem = Vec<u8>;
+
+#[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Debug, TypeInfo)]
+pub enum License<Hash> {
```

Comment:
I am not sure it is a good idea to use a "closed" enumeration for the license type. That would mean a new runtime for each new license which makes it a quite complex process while it should remain simple IMHO.

Reply:
Agreed. Changed in 68c89bf1879e26f1746e24f25cfc59ce01c5d466